src/utils/trusts.ts

```diff
@@ -158,14 +158,16 @@ const getBoIndividualAssignableToTrust = (
   appData: ApplicationData,
 ): BeneficialOwnerIndividual[] => {
   return (appData[BeneficialOwnerIndividualKey] ?? [])
-    .filter((bo: BeneficialOwnerOther) => bo.trustees_nature_of_control_types?.length);
+    .filter((bo: BeneficialOwnerIndividual) => bo.trustees_nature_of_control_types?.length)
+    .filter((bo: BeneficialOwnerIndividual) => Object.keys(bo.ceased_date || {}).length === 0);
```

Comment:
Can you clarify what the **"|| {}"** adds? Is it just covering all non null fields even if they are empty?

Reply:
it's to prevent an error if ceased_date is undefined. If we call .length on an undefined object we'll get the equivalent of a null pointer exception/error. So the || will mean that {} (empty object) gets returned/used if ceased_date is undefined. it's like saying use bo.ceased date if not undefined else use {}.

Reply:
i basically took it from the kainos code that decides whether to show the ceased date on the screen or not for BOs